src/lib/eina/eina_inline_lock_posix.x

```diff
@@ -38,54 +37,18 @@
 # include <pthread.h>
```

Comment:
maybe just one #include<pthread.h> ?

Reply:
So...This is how EFL master those.
You just made me notice that I've deleted a little too much here.  
But the point is, besides this error, I've just copy-pasted it.